### PR TITLE
UI fixes seen in production

### PR DESCRIPTION
**Pull request by pertrai1**
These might not be intended to be seen this way according to the changes
that I am proposing. Seen to be a bit off when I look at the app.

### CODE REVIEW

**Comment by DaleMcGrew** [comment]
This looks great @pertrai1. I will be able to test it out on my local machine in a bit, and then merge it in.

**Comment by DaleMcGrew** [comment]
👍

**Comment by DaleMcGrew** [comment]
@pertrai1 I think this pull request removed the icons from the top navigation -- they are no longer appearing.

**Comment by pertrai1** [comment]
@DaleMcGrew Is this on wevote.me right now? It seems that the icons are there and are not absolute to the right of the window. If this is not on wevote, this commit can be reverted. Most of what I see on the UI seems to be font sizes that are extremely large.

**Comment by DaleMcGrew** [comment]
Hi @pertrai1, I didn't push the changes live to WeVote.me -- I will revert the styles necessary to get the top nav items back. Thank you!

**Comment by pertrai1** [comment]
👍

File: src/sass/components/_itemActionBar.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@
 .item-actionbar {
   display: flex;
   border-top: 1px solid #DDD;
-  margin-top: 1em;
+  margin-top: 1em 0;
```
Comment (ixley) [inline src/sass/components/_itemActionBar.scss]: @pertrai1 did you mean to change `margin-top` to `margin`?

Comment (pertrai1) [inline src/sass/components/_itemActionBar.scss]: @ixley great catch. Did not even see that.

**Pull request merged**